protocol/directory.go

```diff
@@ -39,12 +39,15 @@ func (d *ConiksDirectory) Update() {
 	}
 }
 
-func (d *ConiksDirectory) SetPolicies(epDeadline merkletree.TimeStamp, vrfKey vrf.PrivateKey) {
-	d.policies = merkletree.NewPolicies(epDeadline, vrfKey)
+func (d *ConiksDirectory) UpdatePolicies(epDeadline merkletree.TimeStamp, vrfKey vrf.PrivateKey) {
+	d.policies.Update(epDeadline, vrfKey)
 }
 
+// EpochDeadline returns the EpochDeadline of the current policies
+// If you want to query the EpochDeadline before the new policies
+// is updated, use LatestSTR().Policies.EpochDeadline
```

Comment:
Sorry for not being clear. Just the `UpdatePolicies` stuff.  The shallow copy in `pad.policies = *policies` is all we were after.

Let me update this, one sec.